R/init.R

```diff
@@ -97,6 +97,10 @@ if (interactive() &&
               if (isS4(obj)) {
                 info$slots <- slotNames(obj)
               }
+              if (is.list(obj) ||
+                  is.array(obj)) {
+                info$dim  <- dim(obj)
```

Comment:
I think we could always write `dim` as long as there's `!is.null(dim(obj))`?

Or minimal output, only write `dim` when `is.list(obj) && !is.null(dim(obj))`.

Reply:
I think to be in line with RStudio, we would opt for the minimal output you suggested

Reply:
Sure